rails_event_store/spec/middleware_spec.rb

```diff
@@ -1,42 +1,77 @@
 require 'spec_helper'
+require 'action_controller/railtie'
+require 'rails_event_store/railtie'
+require 'securerandom'
 require 'rails_event_store/middleware'
 require 'rack/lint'
 
 module RailsEventStore
   RSpec.describe Middleware do
-    specify 'lint' do
-      request = ::Rack::MockRequest.new(::Rack::Lint.new(Middleware.new(
-        ->(env) { [200, {}, ['Hello World']] },
-        ->(env) { { kaka: 'dudu' } } )))
+    specify 'calls app within with_metadata block when app has configured the event store instance' do
```

Comment:
Isn't that a bit too much of testing implementation details, something which one would call _klasotesty_?

Reply:
nvmd, changed later